### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by trecouvr**
Hi GitRon,

Some help toward fixing the tests. Everything passes except the Pyhon27 tests because of the requirements in the setup.py :

    django-ses requires Python '>=3.4.8' but the running Python is 2.7.15

### CODE REVIEW

File: tests/test_backend.py [hunk 1]
```diff
@@ -46,96 +45,72 @@ def __call__(self, message, dkim_domain=None, dkim_key=None,
...
+    def setUp(self):
+        # TODO: Fix this -- this is going to cause side effects
+        django_settings.EMAIL_BACKEND = 'tests.test_backend.FakeSESBackend'
+        django_ses.boto3.client = FakeSESConnection
+        self.outbox = FakeSESConnection.outbox
 
     def tearDown(self):
-        # Empty outbox every time test finishes
-        self.fake_connection.outbox = []
+        # Empty outbox everytime test finishes
```
Comment (GitRon) [inline tests/test_backend.py]: you replaced my typo fix :)

File: tests/test_backend.py [hunk 2]
```diff
@@ -46,96 +45,72 @@ def __call__(self, message, dkim_domain=None, dkim_key=None,
...
-                send_mail(subject, body, from_email, recipient_list)
-        self.outbox.append(EmailMultiAlternatives(subject, body, from_email, recipient_list))
+                },
+            }
+        return response
 
 
 class FakeSESBackend(django_ses.SESBackend):
-    """
+    '''
```
Comment (GitRon) [inline tests/test_backend.py]: why did you change by my improvement that PyCharm suggested? just curious...

File: tests/test_backend.py [hunk 3]
```diff
@@ -46,96 +45,72 @@ def __call__(self, message, dkim_domain=None, dkim_key=None,
...
+    def _rfc2047_helper(self, value_to_encode):
         # references: https://docs.python.org/3/library/email.header.html, https://tools.ietf.org/html/rfc2047.html
         name, addr = email.utils.parseaddr(value_to_encode)
         name = email.header.Header(name).encode()
         return email.utils.formataddr((name, addr))
 
     def test_rfc2047_helper(self):
         # Ensures that the underlying email.header library code is encoding as expected, using known values
-        unicode_from_addr = u'Unicode Name Österreich <from@example.com>'
+        unicode_from_addr = u'Unicode Name óóóóóó <from@example.com>'
```
Comment (GitRon) [inline tests/test_backend.py:102]: i chose an actual word with a non-default char, thought it might be more clear what is happening here

**Review by GitRon — CHANGES_REQUESTED** [review]
would be great if you could look at my comments, nothing serious though

**Comment by GitRon** [comment]
> 2.7.15

Fixed it. You could merge my branch in yours again, then everything should pass, right?

File: tests/test_backend.py (hunk 3 above)
Comment (trecouvr) [reply tests/test_backend.py:102]: For the tests I started fresh from the repo master branch, this is why some of your changes were not reflected. I updated the obvious ones you mentioned, but for this one I am not sure. I would rather make the minimal changes to the tests, especially when this change result on different behaviour :

```
AssertionError: '=?utf-8?q?Unicode_Name_=C3=96sterreich?= <from@example.com>' != '=?utf-8?b?VW5pY29kZSBOYW1lIMOzw7PDs8Ozw7PDsw==?= <from@example.com>'
- =?utf-8?q?Unicode_Name_=C3=96sterreich?= <from@example.com>
+ =?utf-8?b?VW5pY29kZSBOYW1lIMOzw7PDs8Ozw7PDsw==?= <from@example.com>
```

You can see that before, the encoding resulting of the tested string was base64 (`VW5pY29kZSBOYW1lIMOzw7PDs8Ozw7PDsw==`), and after your change it's something else (`=C3=`).

**Comment by trecouvr** [comment]
Hi,

Updated the branch on your last commit, now everything is green on travis.

File: tests/test_backend.py (hunk 2 above)
Comment (trecouvr) [reply tests/test_backend.py]: My mistake, I started from a fresh file and forgot some of your changes apparently.

File: tests/test_backend.py (hunk 1 above)
Comment (trecouvr) [reply tests/test_backend.py]: My mistake, I started from a fresh file and forgot some of your changes apparently.

**Review by GitRon — APPROVED** [review]
